src/lib/features/playground/feature-evaluator/client.ts

```diff
@@ -75,6 +75,9 @@ export default class UnleashClient {
             if (parentToggle.dependencies?.length) {
                 return false;
             }
+            if (Boolean(parent.enabled) !== Boolean(parentToggle.enabled)) {
```

Comment:
parent is spec for the parent dependency, parentToggle is the actual toggle from the repository

Reply:
I'm not entirely sure what you mean by "parent is spec". does this mean that the flag is enabled in the environment, but none of the strategies pass (or vice versa)?

Reply:
parent is a dependent flag prerequisite state definition we get from the child flag definition. parentToggle is the actual parent flag definition with all the strategies.